stories/SQFormDatePicker.stories.js

```diff
@@ -46,3 +52,50 @@ export const basicDatePicker = () => {
     </SQForm>
   );
 };
+
+export const onlyEnableFirstDayOfMonthDatePicker = () => {
+  const setDisabledDates = date => {
+    // disable all days EXCEPT first day of the month
+    if (date.date() !== 1) {
```

Comment:
Awesome

Reply:
Yes, this is more accurate. I'll commit this change shortly thanks!
The Pickers iterates over every day in the calendar and calls this function on each day